app/javascript/controllers/profile_tag_category_controller.js

```diff
@@ -0,0 +1,101 @@
+import {Controller} from "@hotwired/stimulus";
+import autoComplete from "@tarekraafat/autocomplete.js";
```

Comment:
@harold93 I don't know this package but I like the description:
Simple autocomplete pure vanilla Javascript library.